docs/prod-host-setup.md

```diff
@@ -1,5 +1,28 @@
 # Production Host Setup Recommendations
 
+## Jailer Configuration
+
+Using Jailer in a production Firecracker deployment is highly recommended,
+it will provide additional security boundaries for the Firecracker.
+The Jailer process applies [cgroup](https://www.kernel.org/doc/Documentation/cgroup-v1/cgroups.txt)
```

Comment:
Please move `[cgroup](https://www.kernel.org/doc/Documentation/cgroup-v1/cgroups.txt)` to a new line. The current line is too long.

Reply:
Done in latest commit. Thanks